content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
 
 Below is an example for taking a snapshot of the keyspace served by
 `$ENDPOINT` to the file `snapshot.db`:
+step1:Take an etcd snapshot backup using the following command
```

Comment:
I'd use an [ordered list](https://www.google.com/search?q=ordered+list+in+Markdown) here.